### ISSUE TITLE
Audio of video still playing after closing/releasing ExoPlayer

### ISSUE THREAD

**Issue by tcudev** [google/ExoPlayer#8208]
### [REQUIRED] Issue description
Audio still playing after completly closing/releasing the player and finishing activity

### [REQUIRED] Reproduction steps
I migrated to newest release and my old close-code didnt worked suddenly, so I tried different implementations but the audio is always there.

My new implementation for closing the PlayerActivity

```
 private void releasePlayer() {
        ima.release();
        ima.setPlayer(null);
        playerView.setPlayer(null);
        player.seekTo(0);
        player.clearVideoSurface();
        player.setPlayWhenReady(false);
        player.stop();
        player.release();

        finish();
    }
```

### [REQUIRED] Link to test content
Will submit via Mail

### [REQUIRED] Version of ExoPlayer being used
r2.12.1

### [REQUIRED] Device(s) and version(s) of Android being used
Android 10 - Galaxy S10

**Comment by kim-vde** [comment]
Could you please send us a link to an audio/video file that has the issue? Is the audio that keeps playing from an ad or from a normal video?

**Comment by tcudev** [comment]
> Could you please send us a link to an audio/video file that has the issue? Is the audio that keeps playing from an ad or from a normal video?

It happens on all links, tried a sample link: https://storage.googleapis.com/gvabox/media/samples/stock.mp4

The audio is from the video itself and not from an ad. If I dont finish the activity (for debugging) the player goes black (video gets cleared), timer does not get reset, it simply stops, but audio is still playing. I need to close the whole app to get rid of the audio.

I also tried it without all the ads and its still not working

I went one release back and it works for now but now ads are not playing :D Think i need to make a new issue


Did you got my email? Is the way of creating the player right? It was really difficult to get to know the new constructor and the DataSpec variable
